Rakefile

```diff
@@ -9,7 +9,10 @@ require "standard/rake"
 Bundler::GemHelper.install_tasks(name: "factory_bot")
 
 desc "Default: run the specs, features, and standard ."
-task default: %w[spec:unit spec:acceptance features standard]
+
+default_tasks = %w[spec:unit spec:acceptance features]
+default_tasks << "standard" if RUBY_VERSION < "3.1.0"
```

Comment:
My concern with this is that as we drop support for older Ruby versions we'll eventually stop running standard unless we remember to change this.

I think we can avoid this if we add a `.standard.yml` with:

```rb
ruby_version: "2.5"
```

and then get rid of the references to the outdated cop by running:

```rb 
bundle exec rake standard:fix
```

We'd probably still want to update the `.standard.yml` as we drop versions of Ruby, but even if we forget to do that at least we're guaranteed to have some version of standard running in CI.

Reply:
Ok, I've done this.